plugins/tiddlywiki/highlight/readme.tid

```diff
@@ -1,6 +1,6 @@
 title: $:/plugins/tiddlywiki/highlight/readme
 
-This plugin provides syntax highlighting of code blocks using v9.18.1 of [[highlight.js|https://github.com/isagalaev/highlight.js]] from Ivan Sagalaev.
+This plugin provides syntax highlighting of code blocks using v11.4.0 of [[highlight.js|https://github.com/isagalaev/highlight.js]] from Ivan Sagalaev. If you need this plugin to work on legacy browsers, you would need to install the "Highlight Legacy" version instead.
```

Comment:
Good stuff – maybe just spell out what is meant by a legacy browser.

Reply:
done